src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -63,6 +63,8 @@ import { makeTestGroup } from '../../../common/framework/test_group.js';
 import { assert } from '../../../common/util/util.js';
 import { GPUTest, TextureTestMixin } from '../../gpu_test.js';
 
+const kFloatTolerance = 0.01;
```

Comment:
Does this need to be `0.01`? Could it be tighter like `0.000001` ?  Is it an 1bit 8snorm tolerance? Maybe a comment?

Reply:
It is f32 tolerance. But based on OpenGL ES spec language, GLSL float highp precision can be < 32bit float. So we just need a small tolerance (strictly the tolerance should be OneULP(X) where X is at each valid values, but we cannot get that.)

The only valid values are `const arbitraryValues = [990, 685, 446, 175];` and `0`.

I will make it stricter and add comments